src/components/TeaPage/TeaPage.js

```diff
@@ -1,48 +1,63 @@
 import { Component } from 'react'
 import CommentsContainer from '../CommentsContainer/CommentsContainer'
-import './TeaPage.css';
+import Error from '../Error/Error'
+import './TeaPage.css'
 
 class TeaPage extends Component {
   constructor() {
     super()
     this.state = {
       tea: {},
-      error: false
+      error: ''
     }
   }
-  
-  componentDidMount() {
-    fetch(`http://localhost:9000/api/v1/teas/${this.props.selectedTea}`)
-      .then( response=> {
-        if (!response.ok) {
-          throw new Error()
-        }
-        return response.json() 
-      })
-      .then( data => this.setState({ tea: data[0] }))
-      .catch( error => {
-        if (error) {
-          this.setState({ error: true })
-        }
-      })
+
+  componentDidMount = () => {
+    const matchingTea = this.props.teas.find( tea => tea.id === this.props.selectedTea)
+    this.setState({tea: matchingTea})
+    console.log(matchingTea)
   }
+  
+  // componentDidMount() {
+  //   fetch(`http://localhost:9000/api/v1/teas/${this.props.selectedTea}`)
+  //     .then( response=> {
+  //       if (!response.ok) {
+  //         throw new Error()
+  //       }
+  //       return response.json() 
+  //     })
+  //     .then( data => this.setState({ tea: data[0] }))
+  //     .catch( error => {
+  //       if (error) {
+  //         this.setState({ error: true })
+  //       }
+  //     })
+  // }
```

Comment:
leftover cleanup?

Reply:
Yes, I am still needing to convert this component into a functional one so this will stay for now.